davenjeru/django-getting-started
# Initiate django app

**What does this PR do?**
Initiates the Django app.

**How can this be manually tested?**
Clone [this](https://github.com/davenjeru/django-getting-started/tree/initiate-django-app) repository and initiate a virtual environment for the project like so:
`virtualenv venv`
`source venv/bin/activate`

Install the requirements from the requirements.txt file like so:
`pip install -r requirements.txt`

Start the Django development server like so:
`python manage.py runserver`

You should see a 'Congratulations' page when you visit the port on the browser.
